### PR TITLE
Fix Ability PopUp freeze

### LINKED ISSUE #2844: Ability Popup Freezes When Using Skill Swap or Wandering Spirit
pokeemerald-1
pokeemerald-0

### PR DESCRIPTION
**Pull request by AlexOn1ine**
## Description
Fixes Ability Pop up freeze when the ability Wandering Spirit was triggered 

## Images
Ability Pop up freeze without the fix:
wandering_spirit_old

After the fix:
wandering_spirit

## Issue(s) that this PR fixes
Fixes #2844

## **Discord contact info**
AlexOnline#2331

### CODE REVIEW

File: data/battle_scripts_1.s [hunk 1]
```diff
@@ -9181,6 +9181,7 @@ BattleScript_WanderingSpiritActivates::
 	destroyabilitypopup
 	pause 40
```
Comment (AsparagusEduardo) [inline data/battle_scripts_1.s:9183]:
```suggestion
```

File: data/battle_scripts_1.s [hunk 2]
```diff
@@ -9181,6 +9181,7 @@ BattleScript_WanderingSpiritActivates::
...
 
+	copybyte gBattlerAbility, gBattlerAttacker
 	setbyte sFIXED_ABILITY_POPUP, TRUE
```
Comment (AsparagusEduardo) [inline data/battle_scripts_1.s:9185]:
```suggestion
	pause 40
	copybyte gBattlerAbility, gBattlerAttacker
	setbyte sFIXED_ABILITY_POPUP, TRUE
```

**Review by AsparagusEduardo — APPROVED** [review]

**Comment by AsparagusEduardo** [comment]
Is this in-game as well, or just the gif messing up? To see if we should make the additional issue.
image

**Comment by AlexOn1ine** [comment]
> Is this in-game as well, or just the gif messing up? To see if we should make the additional issue. image

Yep, this is in-game.

**Pull request merged**